include/atheme/hook.h

```diff
@@ -104,6 +104,14 @@ struct hook_expiry_req
 	int                     do_expire;  // Write zero here to disallow expiry
 };
 
+struct hook_host_request
+{
+        const char *        host;
```

Comment:
Looks like the indentation is off on this one (though it's been that way before the move as well); we might as well take the opportunity to fix it